docs/refs/api/catalogd-webserver.md

```diff
@@ -0,0 +1,88 @@
+```
+
+!!! note
+
+    The value of the `.status.contentURL` field in a `ClusterCatalog` is an arbitrary string value and can change at any time.
+    While there are no guarantees on the exact value of this field, it will always be a URL that resolves successfully for clients using
+    it to make a request from within the cluster.
+
+## Interacting With the Server
+
+### Supported HTTP Methods
+
+The HTTP request methods supported by the catalogd web server are:
+
+- [GET](https://developer.mozilla.org/en-US/docs/Web/HTTP/Methods/GET)
+- [HEAD](https://developer.mozilla.org/en-US/docs/Web/HTTP/Methods/HEAD)
+
+### Response Format
+
+Responses are encoded as a [JSON Lines](https://jsonlines.org/) stream of File-Based Catalog (FBC) [Meta](https://olm.operatorframework.io/docs/reference/file-based-catalogs/#schema) objects delimited by newlines.
+
+??? example "Example JSON-encoded FBC snippet"
+
+    ```json
+    {
+        "schema": "olm.package",
+        "name": "cockroachdb",
+        "defaultChannel": "stable-v6.x",
+    }
+    {
+        "schema": "olm.channel",
+        "name": "stable-v6.x",
+        "package": "cockroachdb",
+        "entries": [
+            {
+                "name": "cockroachdb.v6.0.0",
+                "skipRange": "<6.0.0"
+            }
+        ]
+    }
+    {
+        "schema": "olm.bundle",
+        "name": "cockroachdb.v6.0.0",
+        "package": "cockroachdb",
+        "image": "quay.io/openshift-community-operators/cockroachdb@sha256:d3016b1507515fc7712f9c47fd9082baf9ccb070aaab58ed0ef6e5abdedde8ba",
+        "properties": [
+            {
+                "type": "olm.package",
+                "value": {
+                    "packageName": "cockroachdb",
+                    "version": "6.0.0"
+                }
+            },
+        ],
+    }
+    ```
+
+    Corresponding JSON lines response:
+    ```jsonlines
+    {"schema":"olm.package","name":"cockroachdb","defaultChannel":"stable-v6.x"}
+    {"schema":"olm.channel","name":"stable-v6.x","package":"cockroachdb","entries":[{"name":"cockroachdb.v6.0.0","skipRange":"<6.0.0"}]}
+    {"schema":"olm.bundle","name":"cockroachdb.v6.0.0","package":"cockroachdb","image":"quay.io/openshift-community-operators/cockroachdb@sha256:d3016b1507515fc7712f9c47fd9082baf9ccb070aaab58ed0ef6e5abdedde8ba","properties":[{"type":"olm.package","value":{"packageName":"cockroachdb","version":"6.0.0"}}]}
+    ```
+
+### Compression Support
+
+The `catalogd` web server supports gzip compression of responses, which can significantly reduce associated network traffic.  In order to signal to that the client handles compressed responses, the client must include `Accept-Encoding: gzip` as a header in the HTTP request.
+
+The web server will include a `Content-Encoding: gzip` header in compressed responses.  
+
+!!! note
+
+    Only catalogs that result in a response size greater than 1400 bytes will be compressed
```

Comment:
```suggestion
    Only catalogs that result in a response size greater than 1400 bytes will be compressed.
```

Reply:
Addressed in https://github.com/operator-framework/operator-controller/pull/1331/commits/de1767ea0edfa05872092fc0aee3dda0616bd305